yarn.lock

```diff
@@ -1564,6 +1564,11 @@
   resolved "https://registry.yarnpkg.com/@aws-sdk/types/-/types-3.40.0.tgz#a9d7926fcb9b699bc46be975033559d2293e60d1"
   integrity sha512-KpILcfvRaL88TLvo3SY4OuCCg90SvcNLPyjDwUuBqiOyWODjrKShHtAPJzej4CLp92lofh+ul0UnBfV9Jb/5PA==
 
+"@aws-sdk/types@^3.1.0":
```

Comment:
I don't understand why yarn.lock would be updated

Reply:
please revert this

Reply:
On my local machine @aws-sdk/client-ec2 was causing errors, so I compared the versions and found that main branch uses 3.35.0 and I was on older so I updated the package. I will revert this and pull from the main.

Reply:
reverted